--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -23,12 +23,9 @@
     }),
     includes = ["."],
     linkopts = select({
-        "@platforms//os:osx": [
-            "-lpthread",
-        ],
-        "//conditions:default": [
-            "-lpthread",
+        "@platforms//os:linux": [
             "-lrt",
         ],
+        "//conditions:default": [],
     }),
     visibility = ["//visibility:public"],
